source/acquaman/AMGenericContinuousScanConfiguration.cpp

```diff
@@ -0,0 +1,328 @@
+#include "AMGenericContinuousScanConfiguration.h"
+
+#include "ui/acquaman/AMGenericContinuousScanConfigurationView.h"
+#include "acquaman/AMGenericContinuousScanController.h"
+#include "beamline/AMBeamline.h"
+
+#include <math.h>
+
+AMGenericContinuousScanConfiguration::AMGenericContinuousScanConfiguration(QObject *parent)
+	: AMContinuousScanConfiguration(parent)
+{
+	setName("Generic Continuous Scan");
+	setUserScanName("Generic Continuous Scan");
+}
+
+AMGenericContinuousScanConfiguration::AMGenericContinuousScanConfiguration(const AMGenericContinuousScanConfiguration &original)
+	: AMContinuousScanConfiguration(original)
+{
+	setName(original.name());
+	setUserScanName(original.name());
+	i0_ = original.i0();
```

Comment:
Is i0_ initialized in an ancestor class? I don't think it's initialized in the constructor here.